src/main/java/app/controller/LabelController.java

```diff
@@ -0,0 +1,55 @@
+package app.controller;
+
+import app.dto.LabelRequestDto;
+import app.dto.LabelResponseDto;
+import app.service.LabelService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.Set;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("api/labels")
+@Tag(name = "Label", description = "Labels management")
+public class LabelController {
+    private final LabelService labelService;
+
+    @PostMapping
```

Comment:
You can add response status here.
```suggestion
    @PostMapping
    @ResponseStatus(HttpStatus.CREATED)
```